### PR TITLE
JsonViewer and mailinator added

### PR DESCRIPTION
**Pull request by manishsaraan**
**[http://jsonviewer.stack.hu/]**

**[This is a great tool to view any json data in human readable form. Great for people who create apis and deal with nested json]**

**[http://www.mailinator.com/]**

**[Mailinator is a disposable email service. It is great for developers who want to test their email sending functionality or for quick signup without using your primary email. ]**

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- [x] I have read the [Contribution guidelines](https://github.com/aviaryan/awesome-no-login-web-apps/blob/master/CONTRIBUTING.md) and I am confident that my PR is suitable. 
- [x] This pull request has a descriptive title. For example, `Add [App name]`, not `Update readme.md` or `Added new entries`.
- [x] The app I added is not a duplicate.

### CODE REVIEW

**Comment by manishsaraan** [comment]
I have added two web application links. @aviaryan Please have a look.

File: README.md [hunk 1]
```diff
@@ -250,6 +250,7 @@ Powershell, Redis and lots others. Also has a good collection of IDEs for progra
 * [CSS Matic](https://www.cssmatic.com/) - CSS tools to quickly generate gradients, border radius, noise textures and box shadow and get corresponding CSS or SASS.
 * [Carbon](https://carbon.now.sh) - Create and share beautiful images of your source code.
 * [jsonstore.io](https://www.jsonstore.io/) - jsonstore.io offers free, secured JSON based API endpoints for small projects. It supports common types of HTTP operations like POST, GET, PUT, DELETE etc.
+* [JsonViewer](http://jsonviewer.stack.hu/) - View json in human readable form.
```
Comment (aviaryan) [inline README.md]: This site doesn't support HTTPS. Also, it doesn't look that popular to me. 
Would you mind switching it with https://jsonformatter.curiousconcept.com/ if both do the same thing?

File: README.md [hunk 2]
```diff
@@ -307,6 +308,7 @@ See [plotting graphs examples](https://www.wolframalpha.com/examples/PlottingAnd
 * [Krutidev <--> Unicode](http://wrd.bih.nic.in/font_KtoU.htm) - Convert Krutidev to Unicode (Mangal) and vice versa in one click.
 * [Resume Nation](https://resume-nation.github.io) - An open source resume builder progressive web app which can work offline. It has a few themes to choose from and it can export the resume in PDF form.
 * [Daily Todo](https://dailytodo.org/) -  Simple todo list maker.
+* [Mailinator](http://www.mailinator.com/) - Disposable email service.
```
Comment (aviaryan) [inline README.md]: Please use `https` link.

**Comment by manishsaraan** [comment]
sorry man. This totally slipped from my mind. Will update the pr.

**Comment by manishsaraan** [comment]
@aviaryan I have updated the pr as per your comments but cant figure out why build is failing.
Please have a look. Thanks

**Pull request merged**